src/dmd/dsymbolsem.d

```diff
@@ -333,6 +335,27 @@ private FuncDeclaration buildPostBlit(StructDeclaration sd, Scope* sc)
         sd.members.push(_alias);
         _alias.addMember(sc, sd); // add to symbol table
     }
+
+    if (sd.hasCopyCtor)
+    {
+        // we have user defined postblit, so we prioritize it
+        if (hasUserDefinedPosblit)
+        {
+            sd.hasCopyCtor = false;
+            return xpostblit;
+        }
+        // we have fields with postblits, so print deprecations
+        if (xpostblit && !xpostblit.isDisabled())
+        {
+            deprecation(sd.loc, "`struct %s` has both a user defined copy constructor and fields with postblits.", sd.toChars);
```

Comment:
```suggestion
            deprecation(sd.loc, "`struct %s` implicitly-generated postblit hides copy constructor", sd.toChars);
```

I don't know how to best express this. The original message is fine once you read the changelog entry, but without knowing the context, it's not obvious why it's a problem. Using "hide" (or whatever you feel is suitable) brings the idea that the copy constructor *should have priority*, as we want to eventually get rid of postblit (if that's still the plan ?).